python/tests/test_account.py

```diff
@@ -229,8 +229,8 @@ def test_mute(self, ac1):
         assert not chat.is_muted()
         assert chat.mute(50)
         assert chat.is_muted()
-        assert chat.mute(-51)
-        assert not chat.is_muted()
+        with pytest.raises(ValueError):
+            chat.mute(-51)
```

Comment:
the cffi api wants a duration, not a timestamp. So this throws no error.

Reply:
At least if we forbid it we should still check somehow whether the mute is lifted after it expired.

Reply:
Oh, I see now what you want to test.  Isn't this sufficiently covered by the Rust-level unittest however?  There a time in the past is set and it is checked that it is no longer muted at the current time.  So maybe it's not important to test here?

But while on that subject, why is the Rust API so different from the ffi API?  Rust sets the end time while ffi sets a duration.  Why don't they both behave the same way?

Reply:
> So maybe it's not important to test here?

right, that should be enough - lets merge it then.

> Why don't they both behave the same way?

The idea with having duration in the cffi is that python doesn't need to fiddle around with unix time.